crates/shim-sgx/src/handler/mod.rs

```diff
@@ -264,6 +321,10 @@ impl guest::Handler for Handler<'_> {
         offset: off_t,
     ) -> sallyport::Result<NonNull<c_void>> {
         let addr = addr.map(|v| v.as_ptr() as usize).unwrap_or(0);
+        // FIXME: Replace `PROT_READ` with `PROT_NONE`, once migrating
+        // to the mainline kernel.  `PROT_READ` is just a workaround for the
+        // patch set version 4.
```

Comment:
Can we file an issue for this and note it in the comment?

Reply:
It is possible.

Reply:
There was one already: https://github.com/enarx/enarx/issues/1892

Reply:
Can you mention that in the comment?

Reply:
`// TODO: https://github.com/enarx/enarx/issues/1892`